packages-exp/auth-exp/src/core/strategies/custom_token.ts

```diff
@@ -56,3 +63,54 @@ export async function signInWithCustomToken(
   await authInternal._updateCurrentUser(cred.user);
   return cred;
 }
+
+/**
+ * Sets a custom token provider.
```

Comment:
Maybe something like, "Sets the custom token provider to be used by the given Auth instance."